### ISSUE TITLE
Add the HTML5 loading screens to the Desktop version

### ISSUE THREAD

**Issue by JackXson-Real** [FunkinCrew/Funkin#2550]
### The Suggestion:
The web loading screen should be added to the desktop version as well, or at least make a better transition instead of just an instant cut to the level. It currently looks very jarring and just unfinished. This is something that has always bugged me since I actually really like the web loading screen and love seeing it every time I load a song, but I'm assuming the reason the loading screen isn't in the desktop version is because the loading times on desktop are much faster, and I'm assuming the web version has to download the assets every time a new level is loaded whereas the desktop version already has everything loaded (NOTE: I am not a developer and have no experience with the codebase of this game, so I don't know if that is actually how it works, please let me know if I am wrong). If adding the web loading screen to desktop is not ideal, I recommend at least adding a simple transition so that it looks a little better. I made a video to showcase my idea for a simple cut to black transition:

https://github.com/FunkinCrew/Funkin/assets/32755192/d5f831ac-c790-477e-a8a2-f498d129b693

**Comment by amyspark-ng** [comment]
they probably have some ideas on it but you're totally right

**Comment by EliteMasterEric** [comment]
Part of the reason this hasn't been implemented yet is because the web and desktop builds of the game handle assets very differently! It's definitely something we're looking into though.

**Comment by JackXson-Real** [comment]
> Part of the reason this hasn't been implemented yet is because the web and desktop builds of the game handle assets very differently! It's definitely something we're looking into though.

Yeah that’s basically what I was assuming. Glad it will be implemented at some point

**Comment by EliteMasterEric** [comment]
This change is part of the performance changes currently slated for v0.7. I have a blog post written up about it that we'll probably end up posting later.

**Comment by DustedDustin** [comment]
Waiting…. would be so neat to not have pitch black loading screens on mobile or desktop builds anymore
